packages/material-ui-system/src/style.test.js

```diff
@@ -118,6 +118,32 @@ describe('style', () => {
     });
   });
 
+  const vSpacingWithTheme = style({
```

Comment:
Should we move this variable inside the scope of the test? It doesn't seem that we need to access it outside.

Reply:
Sure, I'll go ahead and move it in then commit.